README.rst

```diff
@@ -1,7 +1,7 @@
 INF200 Exersices:
 =================
 
-FIRST_NAME SURNAME:
+isak finnoy:
```

Comment:
You don't need to make it all lower case here as they are in the project description.
`Isak Finnoy` should be fine.